Pipeline - import and spreadsheet

odscjames opened the pull request.

jarofgreen committed:
pipeline: Improve Import

jarofgreen committed:
pipeline: Move one status field to correct place

jarofgreen committed:
pipeline: levoca importer

ghost commented:
I know the filenames of the 2 pipeline import commands are odd but they will be used once then deleted.

jarofgreen committed:
pipeline: Start building public files and putting in file store

The user will actually see no change until the 2 TODO's in this commit
are done.

But we are doing this now because in dev situations, we can use this to
get data to check easier so it helps us now.

jarofgreen committed:
pipeline: Add drop downs to spreadsheet

jarofgreen committed:
pipeline: Add public spreadsheet
